compress/compress_test.go

```diff
@@ -2,10 +2,29 @@ package compress
 
 import (
 	"fmt"
+	clog "github.com/CortexFoundation/CortexTheseus/log"
+	"github.com/golang/snappy"
 	"log"
 	"testing"
+	"time"
 )
 
+func SnappyEncode(data []byte) ([]byte, error) {
```

Comment:
why move these methods here?

Reply:
it seems that was just used for test

Reply:
> it seems that was just used for test

Oh no , it is the same with zip, another compress method but not used now.

Reply:
it seems that was only used in test

Reply:
https://github.com/golang/snappy

Reply:
You can just add some comment above them to fix the linter

Reply:
ok, i thought it was just to test zip can figure out the same result